# GPIO data

int32 pin
string name
bool value